<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EquipmentRoomWidget</class>
 <widget class="QWidget" name="EquipmentRoomWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>613</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="1,0,0,1,0,2">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="4" column="2">
    <widget class="BallValveWidget" name="wBV_N1" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="BallValveWidget" name="wBV_N3" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="HandValveWidget" name="wHV_N" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="ManometerWidget" name="wPI_N" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>150</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QLabel" name="labelPipesBottom">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Bottom pipes with junction</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLabel" name="labelPipes2">
     <property name="text">
      <string>Two vertical pipes</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="labelPipes1">
     <property name="text">
      <string>Two vertical pipes</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="labelPipesInlets">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Three vertical pipes with mounts</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QLabel" name="labelPipes3">
     <property name="text">
      <string>Three vertical pipes</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="labelMV_N2">
     <property name="text">
      <string>MV N2</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HandValveWidget</class>
   <extends>QWidget</extends>
   <header>handvalvewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BallValveWidget</class>
   <extends>QWidget</extends>
   <header>ballvalvewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ManometerWidget</class>
   <extends>QWidget</extends>
   <header>manometerwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
